src/EventCell.js

```diff
@@ -139,6 +141,15 @@ class EventCell extends React.Component {
                 'rbc-event-continues-prior': continuesPrior,
                 'rbc-event-continues-after': continuesAfter,
               })}
+              onBlur={e => {
+                // https://gist.github.com/pstoica/4323d3e6e37e8a23dd59 - AR Mon Oct 23 10:35:26 EDT 2017
+                const currentTarget = e.currentTarget;
+                setTimeout(() => {
+                  if (!currentTarget.contains(document.activeElement)) {
+                    onSelect({}, e);
+                  }
+                }, 0);
```

Comment:
Not a change request, but you can achieve the same result by omitting the ms parameter 😄 
```js
setTimeout(() => {
  if (!currentTarget.contains(document.activeElement)) {
    onSelect({}, e);
  }
});
```

Reply:
Cool